### DISCUSSION TITLE
Add Sql Server LocalDb support

### DISCUSSION THREAD

**Discussion by mwpowellhtx** [nhibernate/fluent-nhibernate#378] [General] [answered]
Hello,

Don't know if this is a covered use case or not.

I'd like to determine how to connect with an ***MsSqlLocalDb***. Am I able to do so using, for example, [*MsSql2012*](http://github.com/jagregory/fluent-nhibernate/blob/1dff2abe031adf7caeccdbb4c134c3937672cb2b/src/FluentNHibernate/Cfg/Db/MsSqlConfiguration.cs) configuration? Or do I need another [*dialect*](http://github.com/nhibernate/nhibernate-core/blob/af6129e0bd8eca887ab7925c223a6dba7175d201/src/NHibernate/Dialect/MsSql2012Dialect.cs)?

Never mind more up to date Sql Server dialects (i.e. to 2014, or perhaps even 2016, if necessary); however, I'm not in a position to be upgrading those in the near future.

Any suggestions?

**Comment by hazzik** [comment, marked as answer]
MsSql2012 should work.
